xen-bugtool

```diff
@@ -1067,6 +1067,10 @@ exclude those logs from the archive.
                     'vgscan-log']])
     tree_output(CAP_XENSERVER_INSTALL, INSTALLED_REPOS_DIR)
     tree_output(CAP_XENSERVER_INSTALL, UPDATE_APPLIED_DIR)
+    func_output(CAP_XENSERVER_CONFIG, 'snmp_xs_conf', snmp_xs_conf)
+    func_output(CAP_XENSERVER_CONFIG, 'snmpd_xs_conf', snmpd_xs_conf)
+    func_output(CAP_XENSERVER_CONFIG, 'snmpd_conf', snmpd_conf)
+    func_output(CAP_XENSERVER_CONFIG, 'snmpd', snmpd)
```

Comment:
```suggestion
    func_output(CAP_XENSERVER_CONFIG, 'snmp_xs_conf', filter_snmp_xs_conf)
    func_output(CAP_XENSERVER_CONFIG, 'snmpd_xs_conf', filter_snmpd_xs_conf)
    func_output(CAP_XENSERVER_CONFIG, 'snmpd_conf', filter_snmpd_conf)
    file_output(CAP_XENSERVER_CONFIG, [SYSCONFIG_SNMPD])
```
See below, and also add `SYSCONFIG_SNMPD = '/etc/sysconfig/snmpd'`

Reply:
Updated.